fdbserver/Knobs.cpp

```diff
@@ -460,7 +460,8 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( IOPS_UNITS_PER_SAMPLE,                                10000 * 1000 / STORAGE_METRICS_AVERAGE_INTERVAL_PER_KSECONDS / 100 );
 	init( BANDWIDTH_UNITS_PER_SAMPLE,                           SHARD_MIN_BYTES_PER_KSEC / STORAGE_METRICS_AVERAGE_INTERVAL_PER_KSECONDS / 25 );
 	init( BYTES_READ_UNITS_PER_SAMPLE,                          100000 ); // 100K bytes
-	init( EMPTY_READ_PENALTY,                                    20 ); // 20 bytes
+	init( EMPTY_READ_PENALTY,                                   20 ); // 20 bytes
+	init( READ_SAMPLING_ENABLED,                                true ); // enable/disable read sampling
```

Comment:
Please add BUGGIFY for `READ_SAMPLING_ENABLED ` so that we can test in simulation disabling sampling won't cause problem.